include/struct_defs/box_pokemon.h

```diff
@@ -1,22 +1,25 @@
 #ifndef POKEPLATINUM_BOX_POKEMON_H
 #define POKEPLATINUM_BOX_POKEMON_H
 
-#include "struct_defs/pokemon_substruct_1.h"
-#include "struct_defs/pokemon_substruct_2.h"
-#include "struct_defs/pokemon_substruct_3.h"
-#include "struct_defs/pokemon_substruct_0.h"
+#include "struct_defs/pokemon_data_block_b.h"
+#include "struct_defs/pokemon_data_block_c.h"
+#include "struct_defs/pokemon_data_block_d.h"
+#include "struct_defs/pokemon_data_block_a.h"
 
+/**
+ * @brief Boxed Pokemon data structure
+ */
 typedef struct {
```

Comment:
We allow use of both `struct <name>` and `<name>` via the typedef:

```suggestion
typedef struct BoxPokemon {
```